tools/ci/update_pr_preview.py

```diff
@@ -177,6 +178,69 @@ def remove_label(self, pr_number, label_name):
                 'Attempted to remove non-existent label: {}'.format(label_name)
             )
 
+    def create_deployment(self, ref, pr_number):
+        url = '{}/repos/{}/{}/deployments'.format(
+            self.api_root, self.owner, self.repo
+        )
+        data = {
+            'ref': ref,
+            'task': 'deploy',
+            'auto_merge': False,
+            'environment': 'preview',
+            'transient_environment': True
+        }
+
+        logger.info('creating deployment')
+
+        deployment = request(url, 'post', json_data=data)
+
+        if not deployment.id:
+            logger.error('Failed to created deployment')
+            return
+
+        logger.info('Creating status for deployment {}'.format(
+            deployment.id
+        ))
+        url = url + '/{}/statuses'
+        data = {
+            'state': 'success',
+            'environment_url': 'https://wpt.reviews/{}/'.format(
```

Comment:
```suggestion
            'environment_url': 'http://w3c-test.org/submissions/{}'.format(
```

As per https://github.com/web-platform-tests/wpt/pull/18772/files#diff-fad1e181b4cb0da669a8e5a28002069fR49